lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -7,19 +7,19 @@ END_JOURNEY:
 CRI_STATE:
   events:
     next:
-      targetState: ERROR
+      targetState: UNEXPECTED_EVENT
```

Comment:
Is it worth adding a comment here to describe what you should do if this occurs? i.e. 'you must define this route explicitly for each CRI'

I also think it'd be fine to just omit this (especially now we've removed `end`), but being explicit is nice too :)